cli/module_loader.rs

```diff
@@ -317,13 +318,15 @@ pub struct CliModuleLoaderFactory {
 }
 
 impl CliModuleLoaderFactory {
+  #[allow(clippy::too_many_arguments)]
```

Comment:
Yikes

Reply:
These are just constructor dependencies so it's fine.